tools/pipeline_perf_test/test_suites/collector_batch_comparison/test-suite-docker.yaml

```
@@ -17,7 +17,8 @@ components:
         environment:
           - OTLP_ENDPOINT=otel-collector:4317
           - SYSLOG_SERVER=otel-collector
-          - SYSLOG_PORT=514
+          - SYSLOG_PORT=515
```

nit: Use port `514` for consistency.

```suggestion
          - SYSLOG_PORT=514
```